tabular/src/autogluon/tabular/models/tab_transformer/tab_transformer_model.py

```diff
@@ -216,18 +215,19 @@ def tt_fit(self, trainloader, valloader=None, y_val=None, state=None, time_limit
             pretext = pretext_tasks['BERT_pretext'](self.params)
         elif state == 'finetune':
             base_exp_decay = self.params['base_exp_decay']
-            optimizer_fc = optim.Adam(self.model.fc.parameters(), lr=lr, weight_decay=weight_decay)
+            print(type(self.model.embed.parameters()))
```

Comment:
print?

Reply:
This was lingering in the debug for the previous commit fix. Removed it.